### PR TITLE
4657: display statistics on review petition

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2020-05-19 at 9 35 59 AM

### CODE REVIEW

File: shared/src/business/utilities/formatMoney.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * formats a number as money with a dollar sign, commas, and 2 decimal places
+ *
+ * @param {number} number the number to format as money
+ * @returns {string} the formatted string
+ */
+export const formatMoney = number => {
```
Comment (kkoskelin) [inline shared/src/business/utilities/formatMoney.js]: I wonder if you could `export const formatDollars = (new Intl.NumberFormat(....)).format;`

Comment (rachaelparris) [reply shared/src/business/utilities/formatMoney.js]: Yep!

File: web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx [hunk 2]
```diff
@@ -263,6 +263,54 @@ export const ReviewSavedPetition = connect(
                         )}
                       </div>
                     </div>
+                    {reviewSavedPetitionHelper.showStatistics && (
+                      <>
+                        <div className="grid-row grid-gap">
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx:253]: When we are formatting tabular data, it is often best to use actual `table` elements for semantic markup and accessibility.  Screenreaders do a really good job of navigating to a cell in a row of data and stating the column label, but I'm not sure that is done in this instance.

File: web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx [hunk 3]
```diff
@@ -239,17 +239,17 @@ export const ReviewSavedPetition = connect(
...
-                            Type of notice/case
-                          </span>
-                          {form.caseType}
-                        </div>
                       </div>
-                      <div className="tablet:grid-col-4 margin-bottom-1">
+                      <div className="grid-col-4">
+                        <span
+                          className="usa-label usa-label-display"
+                          htmlFor="filing-type"
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx]: I know you didn't start the fire... but `htmlFor` is supposed to reference the ID of a form input. If this element doesn't have that kind of relationship, it should be removed.

File: web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx [hunk 4]
```diff
@@ -263,6 +263,54 @@ export const ReviewSavedPetition = connect(
                         )}
                       </div>
                     </div>
+                    {reviewSavedPetitionHelper.showStatistics && (
+                      <>
+                        <div className="grid-row grid-gap">
+                          <div className="grid-col-4">
+                            <span
+                              className="usa-label usa-label-display"
+                              htmlFor="filing-type"
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx]: same comment here (and below) for `htmlFor` attributes.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx (hunk 4 above)
Comment (rachaelparris) [reply web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx]: This is in this file 8 times now ha - fixing!

File: web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx:253]: Making this a table and getting the styling right is proving difficult, so I've fixed everything else, but I'm going to push this off to another PR.

**Pull request merged**